string                  name
bool                    allow_collision
